src/components/Publish/Pricing/index.tsx

```diff
@@ -23,7 +23,10 @@ export default function PricingFields(): ReactElement {
   function handleTabChange(tabName: string) {
     const type = tabName.toLowerCase()
     setFieldValue('pricing.type', type)
+    setFieldValue('pricing.price', 0)
+    setFieldValue('pricing.freeAgreement', false)
     type === 'dynamic' && setFieldValue('pricing.amountDataToken', 1000)
+    type === 'fixed' && setFieldValue('pricing.amountDataToken', 1000)
```

Comment:
not entirely sure whey we do this but might be a good reason. In any case, it's unneeded duplication if we just turn around the logic with `type !== 'free' && setFieldValue(...)`

Reply:
this is need it to avoid having the `FIXED` tab disabled like this: 

<img width="1804" alt="Screenshot 2022-02-14 at 14 56 19" src="https://user-images.githubusercontent.com/13741335/153877637-7ed260e5-0ab3-45e3-b24f-578f8ad7a561.png">

maybe i misunderstood how this is being handle. But in anyway, your logic is more correct: `type !== 'free' && setFieldValue(...)` should work as well.